rgw/admin/user.go

```diff
@@ -26,7 +26,7 @@ type User struct {
 	UserQuota           QuotaSpec      `json:"user_quota"`
 	TempURLKeys         []interface{}  `json:"temp_url_keys"`
 	Type                string         `json:"type"`
-	MfaIds              []interface{}  `json:"mfa_ids"`
```

Comment:
Oh, damn, I missed this one. We can't change that. That's external API. I will make a fix-up.